content/login/login.html

```diff
@@ -77,19 +77,19 @@ <h1 id="headerTitle">Serify</h1>
         <h1 id="howText">How does it work?</h1>
```

Comment:
What if we just remove this line?  Would that work, would it still be clear what is we are doing.  The more I think about it the more this line is just noise.

On some screen sizes it pushes the flow chart down too much, enough that you can still see it, but too much to read.